packages/manager/src/features/CloudPulse/shared/CloudPulseDashboardSelect.tsx

```diff
@@ -90,6 +90,9 @@ export const CloudPulseDashboardSelect = React.memo(
     }, [dashboardsList]);
     return (
       <Autocomplete
+        errorText={
```

Comment:
Why we want to explicitly check errorText?.length again? may be using a ternary that directly checks if dashboardsList is populated without the Boolean conversion, which is implicit make sense?  Like 
`errorText={dashboardsList?.length ? '' : errorText}`

Reply:
@kmuddapo , agreee updated to errorText={Boolean(dashboardsList?.length) ? '' : errorText}